packages/env-tests/src/scaffold.ts

```diff
@@ -21,22 +36,24 @@ export async function fundAccount(
   })
   context.kit.connection.addAccount(root.privateKey)
 
-  const stableToken = await context.kit.contracts.getStableToken()
+  for (const stableToken of context.stableTokensToTest) {
+    let stableTokenInstance = await initStableTokenFromRegistry(stableToken, context.kit)
 
-  const rootBalance = await stableToken.balanceOf(root.address)
-  if (rootBalance.lte(value)) {
-    logger.error({ rootBalance: rootBalance.toString() }, 'error funding test account')
-    throw new Error(
-      `Root account ${root.address}'s balance (${rootBalance.toPrecision(
-        4
-      )}) is not enough for transferring ${value.toPrecision(4)}`
-    )
-  }
-  const receipt = await stableToken
-    .transfer(recipient.address, value.toString())
-    .sendAndWaitForReceipt({ from: root.address, feeCurrency: stableToken.address })
+    const rootBalance = await stableTokenInstance.balanceOf(root.address)
+    if (rootBalance.lte(value)) {
+      logger.error({ rootBalance: rootBalance.toString() }, 'error funding test account')
```

Comment:
You will want to amend all logging statements with the stabletoken for debugging

Reply:
Done!